### PR TITLE
Fix client WebSocket maxFrameSize and timeout configuration

### PR DESCRIPTION
**Pull request by e5l**
Fix #KTOR-923

### CODE REVIEW

**Comment by Stexxe** [comment]
The test, that will fail before the change and will pass after, is missing.

**Comment by e5l** [comment]
Done, could you take a look?

File: ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import io.ktor.client.features.websocket.*
+import io.ktor.http.cio.websocket.*
+import kotlinx.coroutines.channels.*
+import kotlin.coroutines.*
+import kotlin.test.*
+
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
```
Comment (Stexxe) [inline ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt:8]: Should the license info be on top of the file?

File: ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+class WebSocketTest {
+
+    @Test
+    fun testAsDefault() {
+        val feature = WebSockets(42, 16)
+        val session = object : WebSocketSession {
```
Comment (Stexxe) [inline ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt:16]: What prevents the direct creation of `WebSocketSession`?

**Review by Stexxe — CHANGES_REQUESTED** [review]

File: ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt (hunk 2 above)
Comment (e5l) [reply ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt:16]: It's an interface so we need connected channels to create any `real` instance. So I think it's enough to use mock for the test.

File: ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt (hunk 1 above)
Comment (e5l) [reply ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt]: Sorry, IDEA autoimport. Fixed

**Review by Stexxe — APPROVED** [review]

**Pull request merged**